src/pages/components/filter-panel.mdx

```diff
@@ -0,0 +1,268 @@
+
+<Column colMd={4} colLg={4}>
+
+![Example image: parent tier filter item selected, with child tier items exposed.](../../images/component/filter-panel/filter-panel-single-selection-child-tier.jpg)
+
+<Caption>Example of parent tier filter item selected, with child tier items exposed.</Caption>
+
+</Column>
+
+</Row>
+
+### Multiselect
+Use a multiselect filter when the user has the option to pick more than one attribute to modify the data results. Under the hood, multiselects behave like checkboxes.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: multiselect with no selected filter item.](../../images/component/filter-panel/filter-panel-multiselect-not-selected.jpg)
+
+<Caption>Example of multiselect with no selected filter item.</Caption>
+
+</Column>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: multiselect with multiple filter items selected.](../../images/component/filter-panel/filter-panel-multiselect-selected.jpg)
+
+<Caption>Example of multiselect with multiple filter items selected.</Caption>
+
+</Column>
+
+</Row>
+
+### Multiple categories
+A category is a set of filter items within the same topic. For example, “size” is a category and small, medium, large, and extra large are its filter choices. Multiple filter categories may be applied to the same data set. For example, the user can filter by size, color, and price range.
+
+![A visual example of the Filter panel component with multiple categories in context of a page](../../images/component/filter-panel/filter-panel-multiple-categories.jpg)
+
+## Behaviors
+
+### Larger breakpoints (max, xl, lg)
+The filter panel spans 4 columns of the 16 column grid with a customizable height. It can be configured with any number of filters and categories. 
+
+* The user may expand multiple filter categories at a time.
+* The user may collapse all categories.
+* The user can collapse categories that have filters selected. 
+
+### Scroll
+Filter panel scrolls with the page. Depending on the amount of filters available to the user, the filter panel could exceed the browser height.
+
+### Keyboard interaction
+Filter panel includes a Skip to main content option for keyboard users so they can jump into the main content and/or search results.
+
+### Smaller breakpoints (md and sm)
+On smaller breakpoints, the filter panel is triggered by a button that can be configured anywhere within the experience. Once the user selects the filter button, a modal will open and display the filter panel, the user can close the modal to view the filtered results. The filter button should indicate how many filters the user has selected.
+
+![A visual example of the Filter panel component at a smaller breakpoint](../../images/component/filter-panel/filter-panel-behaviors-smaller-breakpoints.jpg)
+
+<Caption>Example of the Filter panel in smaller breakpoints, the user selects the filter button, selects a number of filter items, and closes the modal to view the results.</Caption>
+
+## Category guidance
```

Comment:
```suggestion
## Default displays
```

Reply:
Jan and I agreed to change this to `Display recommendations`